### PR TITLE
docs: put trailing slash behind matchHost URL

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Use the proper trailing slash behind matchHost URL

## Context:

Based on https://github.com/renovatebot/renovate/discussions/10150#discussioncomment-784396

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/private-npm-modules.md [hunk 1]
```diff
@@ -69,14 +69,16 @@ module.exports = {
       // https://www.jfrog.com/confluence/display/JFROG/npm+Registry
       // Will be passed as `//artifactory.my-company.com/artifactory/api/npm/npm:_auth=<TOKEN>` to `.npmrc`
       hostType: 'npm',
-      matchHost: 'https://artifactory.my-company.com/artifactory/api/npm/npm',
+      matchHost: 'https://artifactory.my-company.com/artifactory/api/npm/npm/',
```
Comment (HonkingGoose) [inline docs/usage/private-npm-modules.md:72]: I think we were missing a trailing slash here before?

File: docs/usage/private-npm-modules.md [hunk 2]
```diff
@@ -69,14 +69,16 @@ module.exports = {
...
       token: process.env.ARTIFACTORY_NPM_TOKEN,
       authType: 'Basic',
     },
   ],
 };
 ```
 
+**NOTE:** Remember to put a trailing slash at the end of your `matchHost` URL, like this: `https://artifactory.my-company.com/artifactory/api/npm/npm/`.
```
Comment (HonkingGoose) [inline docs/usage/private-npm-modules.md:80]:
```suggestion
**NOTE:** Remember to put a trailing slash at the end of your `matchHost` URL.
```

Maybe you want to trim the note?

**Comment by HonkingGoose** [comment]
Our current docs on the `matchHost` property do not mention any trailing slashes, see https://docs.renovatebot.com/configuration-options/#matchhost.

I don't know if **all** `matchHost` URLs need this trailing slash, or only when setting up a specific configuration (self-hosted, private module, etc...). If they all need the trailing slash, I think we should mention that in the property docs as well. 😄 

Please let me know what you like me to do here. 😉

**Comment by viceice** [comment]
> Our current docs on the `matchHost` property do not mention any trailing slashes, see https://docs.renovatebot.com/configuration-options/#matchhost.
> 
> I don't know if **all** `matchHost` URLs need this trailing slash, or only when setting up a specific configuration (self-hosted, private module, etc...). If they all need the trailing slash, I think we should mention that in the property docs as well. 😄
> 
> Please let me know what you like me to do here. 😉

I think this is only required for npm manager / host rules

File: docs/usage/private-npm-modules.md [hunk 3]
```diff
@@ -69,14 +69,16 @@ module.exports = {
       // https://www.jfrog.com/confluence/display/JFROG/npm+Registry
       // Will be passed as `//artifactory.my-company.com/artifactory/api/npm/npm:_auth=<TOKEN>` to `.npmrc`
```
Comment (viceice) [inline docs/usage/private-npm-modules.md:70]:
```suggestion
      // Will be passed as `//artifactory.my-company.com/artifactory/api/npm/npm/:_auth=<TOKEN>` to `.npmrc`
```

The samples there are using trainling slash too. @rarkins maybe we can implicit add the trailing slash too?
image

Comment (rarkins) [reply docs/usage/private-npm-modules.md:70]: But are those examples just for artifactory or apply to every registry?

**Pull request merged**